### DISCUSSION TITLE
RTX 4090

### DISCUSSION THREAD

**Discussion by testmodem** [voicepaw/so-vits-svc-fork#732] [General]
Hello .. 

I am planning to upgrade my gpu to RTX 4090 . This should have no problem with so-vits , right ?

**Comment by TheMeIonGod** [comment]
Unless if you are trying to run a heavy task in the background (Like a Modern AAA game) you should be overkill in fact.

**Comment by akaspeh1** [comment]
> 

https://github.com/pytorch/pytorch/issues/88038
CUFFT_INTERNAL_ERROR on RTX 4090

Had this issue. with pytorch compiled against 11.7 from docker.

Replaced in Dockerfile: From nvidia
```
# FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-runtime
FROM nvcr.io/nvidia/pytorch:23.07-py3
```

No issues so far. Training and infering. Faster that RTX 3090
